examples/quickstart.py

```diff
@@ -0,0 +1,54 @@
+import pathlib
+import pickle
+import tempfile
+
+import stable_baselines3 as sb3
+
+from imitation.algorithms import adversarial, bc
+from imitation.data import rollout
+from imitation.util import logger, util
+
+# Load pickled test demonstrations.
+with open("tests/data/expert_models/cartpole_0/rollouts/final.pkl", "rb") as f:
+    # This is a list of `imitation.data.types.Trajectory`, where
+    # every instance contains observations and actions for a single expert
+    # demonstration.
+    trajectories = pickle.load(f)
+
+# Convert List[types.Trajectory] to an instance of `imitation.data.types.Transitions`.
+# This is a more general dataclass containing unordered
+# (observation, actions, next_observation) transitions.
+transitions = rollout.flatten_trajectories(trajectories)
+
+venv = util.make_vec_env("CartPole-v1")
+
+with tempfile.TemporaryDirectory(prefix="quickstart") as tempdir:
```

Comment:
It is a bit unfortunate that the temporary directory is destroyed when training is finished. We could either be bad and make a temporary directory and not clean it up. Or add some "press enter to exit this script, at which point the result will be deleted" blocking wait at the end.

Reply:
Oops, I forgot this that cleaned up the tmpdir. I think it's fine to leave the tmpdir (it will be auto-cleaned by the system later anyways).